packages/web-sdk/src/instrumentations/performance/navigation.ts

```diff
@@ -1,12 +1,15 @@
 import { genShortID } from '@grafana/faro-core';
 import type { EventsAPI } from '@grafana/faro-core';
+import type {PushEventOptions} from "@grafana/faro-core/src/api/events/types";
 
```

Comment:
`PushEventOptions` is not exported by the main barrel file file (index.ts) in Faro core.

Reply:
done